internal/runbits/runtime/refresh.go

```diff
@@ -40,6 +51,7 @@ func SolveAndUpdate(
 	svcm *model.SvcModel,
 	cfg Configurable,
 	opts Opts,
+	async bool,
```

Comment:
I think you can safely drop this and just use `cfg` to get the async value. Then for the `state refresh` use-case you can check the `trigger`. That should bring the boilerplate further down.

Reply:
This `bool` also serves as an override for the commands that do not respect the config value. We could override the config in the runners but that feels awkward. I'm also not following how checking the trigger can help with that.

Reply:
Never mind... I think I got there on my own 😆